config.py

```diff
@@ -11,6 +11,9 @@
 def bool_config(name):
     return True if os.environ.get(name) == "True" else False
 
+# These should be Slack IDs
+CONTACT_CHANNELS = json.loads(os.environ.get("CONTACT_CHANNELS", "[]"))
```

Comment:
Are we sure this is necessary? We already have a way of disabling commands on a per-channel basis.